src/app/Http/Controllers/UserController.php

```diff
@@ -6,31 +6,87 @@
 
 class UserController extends Controller
 {
-
+    /**
+     * Display the user registration form.
+     *
+     * @return \Illuminate\View\View
+     */
     public function create() {
-
+        return view('users.register');
     }
 
-    public function store() {
+    /**
+     * Process the data submitted from the registration form and create a new user.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function store(Request $request) {
+        $validatedData = $request->validate([
+            'first_name' => 'required|string|max:50',
+            'last_name' => 'required|string|max:50',
+            'email' => 'required|string|email|max:127|unique:users',
+            'username' => 'required|string|max:255|unique:users',
+            'password' => 'required|string|max:255',
+            'user_types' => 'required|string',
```

Comment:
Ten user type v stringu treba previest na UserType enum a tiez zamedzit, aby tam preslo cokolvek ine ako externista/student

Reply:
mozem z formu posielat nieco lepsie, neuvedomila som si to. prerobim to tak, ze tam poslem enum? @Mcibula  @samoslavik45 bude to fungovat, ak dam ako value UserType::EXTERN a UserType::STUDENT?

Reply:
Ako value bude najlepsie asi, ak sa bude posielat `UserType::...->value`, cize `int`, aby sme to mali poprepajane s tym enumom. Kontrolovat sa potom moze, ci `UserType::from(value) === UserType::...`. Bolo by fajn, ak by to bolo centrovane okolo tych enumov.